easypy/concurrency.py

```diff
@@ -574,23 +574,23 @@ def concurrent_map(func, params, workers=None, log_contexts=None, initial_log_in
 class MultiObject(object):
 
     def __init__(self, items=None, log_ctx=None, workers=None, initial_log_interval=None):
-        self._items = list(items) if items else []
+        self._items = tuple(items if items else [])
```

Comment:
`tuple(items) if items else ()`?

Reply:
I agree, though we'd want to search&replace L->T because normally we don't need it as a mutable list.